.github/workflows/build_and_test.yml

```diff
@@ -428,3 +428,41 @@ jobs:
     - name: Build with SBT
       run: |
         ./build/sbt -Pyarn -Pmesos -Pkubernetes -Phive -Phive-thriftserver -Phadoop-cloud -Pkinesis-asl -Phadoop-2.7 compile test:compile
+
+  tpcds-1g:
+    name: Run TPC-DS queries with SF=1
+    runs-on: ubuntu-20.04
+    steps:
+    - name: Checkout Spark repository
+      uses: actions/checkout@v2
+    - name: Checkout TPC-DS (SF=1) generated data repository
+      uses: actions/checkout@v2
+      with:
+        repository: maropu/spark-tpcds-sf-1
```

Comment:
Can we follow https://github.com/apache/spark/pull/31303/files#diff-48c0ee97c53013d18d6bbae44648f7fab9af2e0bf5b0dc1ca761e18ec5c478f2R441-R450? I would be great if we can just stick to using the original repo with addressing https://github.com/apache/spark/pull/31303/files#r563236578. cc @wangyum FYI.

Reply:
Ah, I see. To generate table data for `TPCDSQueryTestSuite`, I think we need more fixes  in the orignal repo because they have different TPC-DS schemas (char/varchar vs string):
spark-sql-perf: https://github.com/databricks/spark-sql-perf/blob/master/src/main/scala/com/databricks/spark/sql/perf/tpcds/TPCDSTables.scala#L57-L542
spark-master/branch-3.1: https://github.com/apache/spark/blob/master/sql/core/src/test/scala/org/apache/spark/sql/TPCDSBase.scala#L51-L548

I've filed a ticket for this: https://github.com/databricks/spark-sql-perf/issues/198 I'll make a PR for it when I have time later.

Reply:
Thanks @maropu. Just to clarify, do you need https://github.com/databricks/spark-sql-perf/pull/196 too?

Reply:
yea, the @wangyum PR looks useful when generating data. (that PR is not a blocker of this PR though)

Reply:
Sure, I will ping him offline to review

Reply:
Thanks, @HyukjinKwon ~